pkg/cli/admin/catalog/build.go

```diff
@@ -48,21 +50,30 @@ type BuildImageOptions struct {
 	FileDir     string
 }
 
-const defaultFromImage = "quay.io/openshift/origin-operator-registry:latest"
+const defaultFromImage = "registry.redhat.io/openshift4/ose-operator-registry"
 
 func NewBuildImageOptions(streams genericclioptions.IOStreams) *BuildImageOptions {
-	// Adjust default build opts to use an OpenShift base "from" image
-	defaults := appregistry.DefaultAppregistryBuildOptions()
-	defaults.From = defaultFromImage
-
 	return &BuildImageOptions{
-		AppregistryBuildOptions: defaults,
+		AppregistryBuildOptions: appregistry.DefaultAppregistryBuildOptions(),
 		IOStreams:               streams,
 		ParallelOptions:         imagemanifest.ParallelOptions{MaxPerRegistry: 4},
 	}
 }
 
 func (o *BuildImageOptions) Complete(cmd *cobra.Command, args []string) error {
+	// default the base image os to linux/amd64 (the most common case)
+	pattern := o.FilterOptions.FilterByOS
+	if len(pattern) == 0 {
+		o.FilterOptions.FilterByOS = regexp.QuoteMeta(fmt.Sprintf("%s/%s", "linux", "amd64"))
+	}
+
+	// default the base image tag to match the version of oc being used
+	_, reportedVersion, _ := version.ExtractVersion()
+	if len(reportedVersion) > 0 {
+		o.From = defaultFromImage + ":v" + reportedVersion
```

Comment:
Experimentally, this seems to do what we want:

```
$ oc version --client
Client Version: 4.3.5
```
corresponds to:
https://catalog.redhat.com/software/containers/detail/5cddd0bed70cc57c44b2e1f3?tag=v4.3.5&architecture=amd64

Reply:
Changed to pull from a release instead.